lib/tabulo/table.rb

```diff
@@ -695,8 +705,7 @@ def normalize_column_label(label)
     end
 
     # @!visibility private
-    def expand_to(min_table_width)
-      columns = get_columns
+    def expand_to(min_table_width, columns: get_columns)
```

Comment:
I don't think there's much value in providing `columns:` with a default value here, since the method is private.

If and when these methods are made public, we can then think about a default value.

Reply:
OK, I've removed the default. I'd added it just to be extra nice in case someone used these methods via `send`.